linker/NetCore.props

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <RuntimeFrameworkVersion>2.0.0-beta-001509-00</RuntimeFrameworkVersion>
     <DefineConstants>$(DefineConstants);NET_CORE</DefineConstants>
```

Comment:
Renamed to `FEATURE_ILLINK` - thanks. I also renamed NetCore.props, and the project configurations (currently netcore_Debug/netcore_Release).